include/packager/hls_params.h

```diff
@@ -63,6 +63,9 @@ struct HlsParams {
   /// Custom EXT-X-MEDIA-SEQUENCE value to allow continuous media playback
   /// across packager restarts. See #691 for details.
   uint32_t media_sequence_number = 0;
+  /// Force the muxer to order streams in the order given
+  /// on the command-line
+  bool force_cl_index = false;
```

Comment:
set to true by default

Reply:
yes, these bool values should also be set to 'true'.